src/main/webapp/app/exam/participate/summary/result-overview/exam-result-overview.component.html

```diff
@@ -2,125 +2,127 @@
-                    </th>
-                    <th>
-                        {{ maxPoints }}
-                    </th>
-                    <th>
-                        <span
-                            *ngIf="studentExamWithGrade.studentResult.overallScoreAchieved !== undefined; else totalScoreNotDefinedPlaceholder"
-                            [class]="hasPassed ? 'text-success' : 'text-danger'"
-                        >
-                            {{ overallAchievedPercentageRoundedByCourseSettings }}%
-                        </span>
-                        <ng-template #totalScoreNotDefinedPlaceholder>
-                            <span>-</span>
-                        </ng-template>
-                    </th>
-                    <th *ngIf="studentExamWithGrade?.maxBonusPoints">
-                        {{ studentExamWithGrade?.maxBonusPoints }}
-                    </th>
-                </tr>
-            </tfoot>
-        </table>
-        <div class="d-flex align-items-center justify-content-evenly">
-            <div>
-                <h5 *ngIf="studentExamWithGrade?.maxBonusPoints">
-                    {{
-                        'artemisApp.exam.examSummary.points.youAchievedWithBonus'
-                            | artemisTranslate
-                                : {
-                                      achievedPoints: overallAchievedPoints,
-                                      normalPoints: maxPoints
-                                  }
-                    }}
-                </h5>
-                <h5 *ngIf="studentExamWithGrade?.maxBonusPoints">
-                    {{
-                        'artemisApp.exam.examSummary.points.youAchieved'
-                            | artemisTranslate
-                                : {
-                                      achievedPoints: overallAchievedPoints,
-                                      normalPoints: maxPoints
-                                  }
-                    }}
-                </h5>
-                <h5 *ngIf="studentExamWithGrade.studentResult.gradeWithBonus != undefined">
-                    {{
-                        'artemisApp.exam.examSummary.points.youAchievedFromBonus.' + studentExamWithGrade.studentResult.gradeWithBonus.bonusStrategy
-                            | artemisTranslate
-                                : {
-                                      achievedBonus: studentExamWithGrade.studentResult.gradeWithBonus.bonusGrade,
-                                      bonusFromTitle: studentExamWithGrade.studentResult.gradeWithBonus.bonusFromTitle
-                                  }
-                    }}
-                </h5>
+    <div class="exam-points-summary-container" [class.d-flex]="displayGradeKeysBesideTable">
+        <div class="grade-results-table-container">
+            <table class="table table-striped table-hover w-auto" id="result-overview-table">
+                <thead>
+                    <tr class="text-nowrap">
```

Comment:
This part was changed a bit (wrapped in a container soo the grading keys can be displayed at the right side if enough space)